packages/test-runner-intercept/README.md

```diff
@@ -0,0 +1,78 @@
+## Usage
+
+Setup
+
+```js
+// web-test-runner.config.mjs
+import { interceptModulePlugin } from '@web/test-runner-intercept/plugin';
```

Comment:
I think we don't have a convention to export plugins for `/plugin`. I'd stick to just the module name

but I also have a question about the module name: the plugin is focused on module interception, so cutting the name to `@web/test-runner-intercept` seems like a bad choice since the term `interceptor` can mean anything and often used in the context of AJAX, so I'd rather give the module the full name `@web/test-runner-intercept-module` or smth like that

Reply:
Actually, we do. In `@web/test-runner-commands` we have this:
![image](https://github.com/modernweb-dev/web/assets/17364880/fded40fb-243c-4894-982d-74643c3a6175)
As this project only has 1 plugin, I used the singular form. The default entry point is used for the browser.

For the name, I tried to make it sweet and short, like other names in the repo: `mocks`, `test-runner-commands`. Both of them also don't mention more context. (ie. are mocks the mocks for wtr or something else; how does commands relate to wtr)

But as such, if more people think it should change, it definitely can, maybe `@web/test-runner-module-interceptor` to stay in line with other plugin names in the repo? `...-loader`, `...-reporter`.

Reply:
I'd say a convention is smth used across all packages, which I checked and found it only in the example you posted, so it seems more of a choice and majority chose to have a single entry point, except where a separate one is required by other tools (e.g. `preset.js` is needed by Storybook or we have some browser-specific entry points)

in your case it's unclear to me how it benefits users?

take for example this exception:

```js
  } catch (e) {
    throw new Error(
      `Module interception is not active. Make sure the \`interceptModulePlugin\` of \`@web/test-runner-intercept\` is added to the Test Runner config.`,
    );
  }
```

I'll need to read docs to understand where to import it from

so for simplicity I'd not introduce extra entry points unless really necessary

Reply:
naming wise, I think we the following package convention

```js
@web/%tool-name%-%plugin-name%
```

so both `@web/test-runner-intercept` and `@web/test-runner-intercept-modules` or even `@web/test-runner-module-interceptor` fit it, jus the question is what describes better the plugin name part

Reply:
> I'd say a convention is smth used across all packages, which I checked and found it only in the example you posted, so it seems more of a choice and majority chose to have a single entry point, except where a separate one is required by other tools (e.g. preset.js is needed by Storybook or we have some browser-specific entry points)
I'm not sure the reason of two entry points are clear. So, I'll explain why it's split. One entry point is for the browser (the default entry point). In this entry points I cannot add the plugin in there, as it would import modules with server side code in the browser. The second one, obviously now, is for the server side code only.
It's not the only package doing this. `@web/test-runner-core` is also splitting their exports in two (but actually 3) entries. And the browser split is also the reason of doing it here.

Reply:
not sure where we are on this right now
IIRC, there were suggestion to use `mock` in some way in all names including package name
I personally think the calling this mock is not correct since this plugin doesn't create mocks, it enables mocking
so you can play with names like `enable-module-mocks` or smth like that, but I don't want to force you too :)
what I do want to improve is the genericness of the name, the `-intercept` is not sufficient, because it's not just a generic intercept and as such can be really confused with AJAX

Reply:
I agree with Mikhail, Im still not convinced that `-intercept` adequately describes what this does. _Technically_ the plugin does _intercept_ the module, but that's not really the _point_ of this project, the point is to make modules mockable/overrideable. So maybe it should be more something like `@web/test-runner-mockable-modules` or something like that

Reply:
Can you resolve this conversation if it's ok? 😃